src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -637,6 +637,16 @@ public static XmlSerializer[] FromTypes(Type[] types)
         {
             if (types == null)
                 return Array.Empty<XmlSerializer>();
+
+#if NET_NATIVE
+            var serializers = new List<XmlSerializer>();
```

Comment:
Good suggestion. I opened issue https://github.com/dotnet/corefx/issues/5389 for tracking feedbacks from here.